pyproject.toml

```diff
@@ -83,22 +97,32 @@ authorized_licenses = [
 ]
 unauthorized_licenses = ["gpl v3"]
 
-[tool.poe.tasks]
-_codestyle_isort = "isort --check-only --diff ."
-_codestyle_black = "black --check --diff ."
-codestyle = ["_codestyle_isort", "_codestyle_black"]
-_format_autoflake = "autoflake --in-place -r ."
-_format_isort = " isort ."
-_format_black = "black ."
-format = ["_format_autoflake", "_format_isort", "_format_black"]
-typecheck = "mypy --install-types --non-interactive --ignore-missing-imports --check-untyped-defs --warn-unused-ignores --show-error-codes ."
-_liccheck_export = "poetry export -f requirements.txt -o .requirements.txt"
-_liccheck_verify = "liccheck -r .requirements.txt"
-liccheck = ["_liccheck_export", "_liccheck_verify"]
-check = ["codestyle", "typecheck", "liccheck"]
-unittest = "pytest --cov --cov-report html --cov-report term  -sv tests"
-test = ["unittest", "check"]
+[tool.autoflake]
+recursive = true
+in-place = true
+remove-all-unused-imports = true
+ignore-init-module-imports = true
 
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+[tool.flake8]
+max-line-length = 88
```

Comment:
as mentioned on Discord, if we'd like to have this unified, better settle to `100` since that's what `yapapi` has been using